### ISSUE TITLE
Index.html has failed Accessibility Checker

### ISSUE THREAD

**Issue by TMerco** [TMerco/github-issue-templates#6]
Describe the bug
The index.html file fails accessibility Checker, the Heading elements aren't in sequentially-descending order and could put us at risk of accessibility lawsuits.

To Reproduce

Go to 'https://www.accessibilitychecker.org/'
Paste in https://github.com/TMerco/github-issue-templates
Click submit
View errors
Expected behavior
Accessibility Checker should show no errors on page

Screenshots
Click on the following link to view the current Accessibility Checker error:

https://www.accessibilitychecker.org/audit/?website=https%3A%2F%2Fgithub.com%2FTMercoV%2Fgithub-issue-templates&flag=us

Desktop (please complete the following information):

N/A
Smartphone (please complete the following information):
N/A
Additional context
Consider using the Accessibility Checker to check for other errors or issues.
